doc/Cifar10.rst

```diff
@@ -0,0 +1,1124 @@
+Estimating prediction sets on the Cifar10 dataset
+=================================================
+
+What is done in this tutorial ?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+   -  **Cifar10 dataset** : 10 classes (horse, dog, cat, frog, deer,
+      bird, airplane, truck, ship, automobile)
+
+..
+
+   -  Use ``mapie.classification.MapieClassifier`` to compare the
+      prediction sets estimated by several conformal methods on the
+      Cifar10 dataset.
+
+   -  Train a small CNN to predict the image classe
+
+..
+
+   -  Create a custom classe ``TensorflowToMapie`` to resolve adherence
+      problems between Tensorflow and Mapie
+
+Tutorial preparation
+--------------------
+
+.. code-block:: python
+
+    import random
+    from typing import Dict, List, Tuple, Union
+    
+    import cv2
+    import matplotlib.pyplot as plt
+    import numpy as np
+    import pandas as pd
+    import tensorflow as tf
+    import tensorflow.keras as tfk
+    from tensorflow.keras.callbacks import EarlyStopping
+    from tensorflow.keras import Sequential
+    from tensorflow.keras.layers import Conv2D, Dense, Dropout, Flatten, MaxPooling2D
+    from tensorflow.keras.losses import CategoricalCrossentropy
+    from tensorflow.keras.optimizers import Adam
+    import tensorflow_datasets as tfds
+    from sklearn.metrics import accuracy_score
+    from sklearn.metrics._plot.confusion_matrix import ConfusionMatrixDisplay
+    from sklearn.model_selection import train_test_split
+    from sklearn.preprocessing import label_binarize
+    
+    from mapie.metrics import classification_coverage_score
+    from mapie.classification import MapieClassifier
+    
+    %load_ext autoreload
+    %autoreload 2
+    %matplotlib inline
+    %load_ext pycodestyle_magic
+
+.. code-block:: python
+
+    SPACE_BETWEEN_LABELS = 2.5
+    SPACE_IN_SUBPLOTS = 4.0
+    FONT_SIZE = 18
+
+
+1. Data loading
+---------------
+
+The Cifar10 dataset is downloaded from the ``Tensorflow Datasets``
+library. The training set is then splitted into a training, validation
+and a calibration set which will be used as follow: > - **Training
```

Comment:
Add a double backspace to allow the bullet points to rend in the documentation.

Reply:
done